dm/tests/gbk/run.sh

```diff
@@ -67,6 +67,105 @@ function run() {
 	check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
 
 	test_from_tidb
+
+	echo "prepare data for invalid connection test"
+	run_sql_source1 "CREATE TABLE IF NOT EXISTS gbk.invalid_conn_test1(i TINYINT, j INT UNIQUE KEY, k TINYINT, m TINYINT, n TINYINT)"
+	run_sql_source1 "CREATE TABLE IF NOT EXISTS gbk.invalid_conn_test2(i TINYINT, j INT UNIQUE KEY, k TINYINT, m TINYINT, n TINYINT)"
+
+	kill_dm_worker
+
+	# test invalid connection with status synced
```

Comment:
I came up with two simple tests that don't need to inject DDL status:

1. inject TestHandleSpecialDDLError, start DM-worker, and execute **one** DDL and insert some data. DM-worker will see injected error and run into `waitAsyncDDL`. Then it will see the DDL is "synced" so replication continues, the data is soon replicated and we can use sync-diff to check the data.
2. inject TestHandleSpecialDDLError, start DM-worker, make a column contains duplicated data and execute **one** DDL that add UNIQUE on the column. DM-worker will see the DDL is "rollbacked" and the task should be paused with ErrCancelledDDL. Then we use `run_dmctl_with_retry` to check the result of query-status.

Reply:
I have put the code for checking whether count is equal to 1 to run.sh instead of count_contains, because count_contains is meant to check the number of appearances instead of justifying whether the count is equal to a certain number. This part should be done in run.sh that call this function.